spec/index.bs

```diff
@@ -4835,56 +4837,115 @@ dictionary GPUComputePassDescriptor : GPUObjectDescriptorBase {
 <dl dfn-type=method dfn-for=GPUComputePassEncoder>
     : <dfn>setPipeline(pipeline)</dfn>
     ::
+        Sets the current {{GPUComputePipeline}}.
 
         <div algorithm="GPUComputePassEncoder.setPipeline">
             **Called on:** {{GPUComputePassEncoder}} this.
 
             **Arguments:**
             <pre class=argumentdef for="GPUComputePassEncoder/setPipeline(pipeline)">
-                pipeline:
+                |pipeline|: The compute pipeline to use for subsequent dispatch commands.
             </pre>
 
-            **Returns:** void
+            **Returns:** `void`
 
-            Issue: Describe {{GPUComputePassEncoder/setPipeline()}} algorithm steps.
+            Issue the following steps on the [=Device timeline=] of |this|.{{GPUObjectBase/[[device]]}}:
+            <div class=device-timeline>
+                1. If any of the following conditions are unsatisfied, make |this| [=invalid=] and stop.
+                    <div class=validusage>
+                        - |pipeline| is [$valid to use with$] |this|.
```

Comment:
I think compute pipelines are alaways valid in compute passes.

Reply:
Even if it was created with another device?

Reply:
Ah sorry I had forgotten what the `valid to use with` is.